pax-compiler/src/parsing.rs

```diff
@@ -302,6 +331,37 @@ fn recurse_pratt_parse_to_string<'a>(
         .parse(expression)
 }
 
+fn parse_rgb_color(color: Pair<'_, Rule>, color_format: &str) -> String {
+    let mut inner = color.into_inner();
+
+    let r = inner.next().unwrap().as_str().replace("%", "");
+    let g = inner.next().unwrap().as_str().replace("%", "");
+    let b = inner.next().unwrap().as_str().replace("%", "");
+    let a = inner
+        .next()
+        .map(|x| x.as_str().replace("%", ""))
+        .unwrap_or("1".to_string());
+
+    let r = format!("Size::Percent({}.into())", r);
+    let g = format!("Size::Percent({}.into())", g);
+    let b = format!("Size::Percent({}.into())", b);
+    let a = format!("Size::Percent({}.into())", a);
+
+    let comma_count = color_format.chars().filter(|&c| c == ',').count();
+
+    if color_format.starts_with("rgb(") && comma_count == 2 {
+        format!("Color::rgb({r},{g},{b}).into()")
+    } else if color_format.starts_with("rgba(") && comma_count == 3 {
+        format!("Color::rgba({r},{g},{b},{a}).into()")
+    } else if color_format.starts_with("hlc(") && comma_count == 2 {
+        format!("Color::hlc({r},{g},{b}).into()")
```

Comment:
I'm not sure this code path is being hit.  When I try `fill={hlc((ticks + i * 360.0 * 3)%, 50%, 50%)}` in the Fireworks example, I get an error:

```
168 | ...   hlc((Size::Percent((ticks +((i *(Numeric::from(360.0 )).into())*(Numer...
    |       ^^^ not found in this scope
```

My best guess from a quick look is that we might be short-circuiting this on line 211 of this file: https://github.com/pax-lang/pax/pull/71/files#diff-3bfd18387cf178bec2da888bfa3aadadd3de4522ed242e2b303877b0a3dcb855R211 , but the issue might be something else entirely.